pkg/broker/types.go

```diff
@@ -104,11 +105,11 @@ type DataStore interface {
 type GetAwsSession func(keyid string, secretkey string, region string, accountId string, profile string, params map[string]string) *session.Session
 
 type GetCfnClient func(sess *session.Session) CfnClient
-type GetSsmClient func(sess *session.Session) *ssm.SSM
+type GetSsmClient func(sess *session.Session) ssmiface.SSMAPI
```

Comment:
I updated these to return the interface so that we can drop in mock implementations.